rules/BUILD.java_runtime.tpl

```diff
@@ -0,0 +1,25 @@
+# Copyright 2016 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file is auto-generated by an rbe_autoconfig repository rule
+# and should not be modified directly.
+# See @bazel_toolchains//rules:rbe_repo.bzl
+
+package(default_visibility = ["//visibility:public"])
+
+java_runtime(
+    name = "jdk",
+    srcs = [],
+    java_home = "%{java_home}",
```

Comment:
If users do nothing, it will be fully based on the JAVA_HOME we set in the container. The currently defined JDK migration plan would not work very well this. We need to think through that in this new config world.

Reply:
Agree (created a bug to track internally)